### ISSUE TITLE
Write a chapter about deploying Tide applications

### ISSUE THREAD

**Issue by mendelt** [http-rs/tide-book#10]
As suggested in this [issue](https://github.com/http-rs/tide/issues/792) in the `Tide` repository we should write some guidance on how to deploy a tide application to production.

**Comment by pepoviola** [comment]
Hi @mendelt, I just see this post https://brokenco.de/2021/02/16/async-h1-error-tide.html and we may want to make a note that point that as this moment `Tide` only support` HTTP 1.1`.

Thx!

**Comment by rtyler** [comment]
I'm not sure I would listen to that guy, his code is all broken! :smiling_imp:

**Comment by mendelt** [comment]
Good idea, and interesting that you mention this. I encountered this issue last week https://github.com/http-rs/tide/issues/470 with a similar remark.
Not sure what happened with this pr though https://github.com/http-rs/tide/pull/467 it should have documented this in the tide README.md but that documentation seems to have disappeared again.

How far along is HTTP 1.0 support? I think this was being worked on in async-h1 right?

**Comment by mendelt** [comment]
@rtyler thanks for that blogpost. Is it ok if i'm going to steal some of that for this chapter?

**Comment by rtyler** [comment]
@mendelt Please do! nginx is now the reverse proxy for https://github.com/rtyler/dotdotvote/ and if you want any other code snippets let me know!

**Comment by jbr** [comment]
we may also want to reinstate something like https://github.com/http-rs/tide/commit/85ecc3633cf1e6f5a8bf955ad1b03090b1d8e621#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759 in the rustdocs, i'm not sure why that went away. i figure anything essential probably should be in both the book and also the docs

**Comment by pepoviola** [comment]
Yes, I think we can add an example configuration for `nginx`,  `apache` and `haproxy`.
